src/SprykerEco/Zed/AmazonQuicksight/Business/AmazonQuicksightFacadeInterface.php

```diff
@@ -47,4 +49,38 @@ public function expandUserCollectionWithQuicksightUsers(
     public function createQuicksightUsersForUserCollectionResponse(
         UserCollectionResponseTransfer $userCollectionResponseTransfer
     ): UserCollectionResponseTransfer;
+
+    /**
+     * Specification:
+     * - Requires `AnalyticsEmbedUrlRequestTransfer.user` and `AnalyticsEmbedUrlRequestTransfer.user.idUser` to be set.
+     * - Returns `true` if Quicksight user with the provided user ID exists in DB.
+     * - Returns `false` otherwise.
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\AnalyticsEmbedUrlRequestTransfer $analyticsEmbedUrlRequestTransfer
+     *
+     * @return bool
+     */
+    public function isQuicksightAnalyticsEmbedUrlProviderApplicable(
+        AnalyticsEmbedUrlRequestTransfer $analyticsEmbedUrlRequestTransfer
+    ): bool;
+
+    /**
+     * Specification:
+     * - Requires `AnalyticsEmbedUrlRequestTransfer.user`, `AnalyticsEmbedUrlRequestTransfer.user.quicksightUser`
+     * and `AnalyticsEmbedUrlRequestTransfer.user.quicksightUser.arn` to be set.
+     * - Sends request to AWS API to generate an embed URL for a registered Quicksight user. For more information see {@link https://docs.aws.amazon.com/quicksight/latest/APIReference/API_GenerateEmbedUrlForRegisteredUser.html}.
+     * - Adds errors to `AnalyticsEmbedUrlResponseTransfer.errors` if Quicksight embed URL generation failed.
+     * - Populates `AnalyticsEmbedUrlResponseTransfer.embedUrl.url` with the generated embed URL.
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\AnalyticsEmbedUrlRequestTransfer $analyticsEmbedUrlRequestTransfer
+     *
+     * @return \Generated\Shared\Transfer\AnalyticsEmbedUrlResponseTransfer
+     */
+    public function getAnalyticsEmbedUrl(
```

Comment:
We have some kind of inconsistency here: you use `QuicksightAnalytics` in hte previous method and `Analytics` is this one.